# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../dietlibc/clearenv.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

A clearenv, as needed for latest u/dev - sigh.

  - Rene Rebe <rene@exactcode.de>

--- dietlibc-0.29/libugly/setenv.c.orig	2005-11-27 00:01:20.000000000 +0100
+++ dietlibc-0.29/libugly/setenv.c	2005-11-27 00:02:27.000000000 +0100
@@ -17,3 +17,8 @@
     return putenv(c);
   }
 }
+
+int clearenv(void) {
+  environ=realloc(environ,1);
+  environ[0]=0;
+}
